specs/prometheus/rule.yaml

```diff
@@ -21,4 +21,135 @@ spec:
       labels:
         issue: Application pod {{$labels.exported_pod}} in namespace {{$labels.namespace}}
           is not hyperconverged
+        severity: warning
+    - alert: ApplicationBackupFailed
+      annotations:
+        message: Unable to backup resources in {{ $labels.namespace
+          }} namespace using application backup {{ $labels.name }}
+        summary: Application backup {{$labels.name}} in namespace {{$labels.namespace}}
+          is failed
+      expr: application_backup_status == 3
+      for: 5m
+      labels:
+        issue: Application backup {{$labels.name}} in namespace {{$labels.namespace}}
+          is failed
+        severity: warning
+    - alert: ApplicationCloneFailed
+      annotations:
+        message: Application clone {{ $labels.name }} in namespace {{ $labels.namespace
+          }} is failed
+        summary: Application clone {{$labels.name}} in namespace {{$labels.namespace}}
+          is failed
+      expr: application_clone_status == 3
+      for: 5m
+      labels:
+        issue: Application clone {{$labels.name}} in namespace {{$labels.namespace}}
+          is failed
+        severity: warning
+    - alert: ApplicationRestoreFailed
+      annotations:
+        message: Application restore {{ $labels.name }} in namespace {{ $labels.namespace
+          }} is failed
+        summary: Application restore {{$labels.name}} in namespace {{$labels.namespace}}
+          is failed
+      expr: application_restore_status == 3
+      for: 5m
+      labels:
+        issue: Application restore {{$labels.name}} in namespace {{$labels.namespace}}
+          is failed
+        severity: warning
+    - alert: ApplicationRestorePartialSuccessful
+      annotations:
+        message: Error while appliying k8s resources for application restore {{ $labels.name }} in namespace {{$labels.namespace}}
+        summary: Application restore {{$labels.name}} in namespace {{$labels.namespace}}
+          is partially successful
+      expr: application_restore_status == 3
+      for: 5m
+      labels:
+        issue: Application restore {{$labels.name}} in namespace {{$labels.namespace}}
```

Comment:
Lets keep the issue as `Application restore <> in namespace <> has partially succeeded`